### PR TITLE
Add contract suspension worked example

### LINKED ISSUE #758: Worked example (1.2): Contract suspension
In the [ageops](https://ageops.net) system in Afghanistan contracts can have status = suspended ([example](https://ageops.net/en/procurement-procedure/contract/progress/RRD%252FMOF%252FNRAP%252FKBL%252FLGR%252F055%252FC2%252F001))

This indicates that the contract is temporarily suspended, for example due to security reasons, and will either be resumed at a later date or cancelled. The system does not capture a reason or description for the suspension.

In OCDS, contract status is a closed codelist with the following codes: pending, active, cancelled, terminated.

Similar to the discussion on suspended tenders in https://github.com/open-contracting/standard/issues/430, there are two main options we could pursue:

**(1) Adding an entry to the contract status codelist**

A contract status of 'suspended' could be added to the codelist.

This would mean that:

* When the contract was suspended, it would cease to show up in lists of active contracts
* Consuming applications would need to be aware of the 'suspended' contract status

Contract status is a closed codelist, which means that any changes would have to be made as part of an upgrade to the standard.

**(2) Adding a new field via an extension**

A true/false `suspended` field could be added to the contract object to indicate whether the contract is suspended

This would mean that:

* Applications that are not aware of the new field would continue to display the contract in a list of active contracts;
* Applications that are aware of the new field could use this to filter views or display extra information about a contract

## Questions

Assuming we pursue option (2), at least until the next standard upgrade:

(1) Do any other systems or legal frameworks use the concept of a suspended contract?

(2) Does a suspended contract fit the description of the `active` code from the contract status codelist:

> This contract has been signed by all the parties, and is now legally in force.

(3) Is 'suspended' the appropriate terminology?

(4) Should an extension also introduce a free text `statusDetails` field where details of the reason for the suspension could be provided.

(5) Should an extension also introduce any 'documentType' codelist entries to link to related reasons for suspension or documentation of the suspension of the contract.

(6) Should guidance include any information on including milestones or other free-text descriptions about suspensions?

### PR DESCRIPTION
**Pull request by duncandewhurst**
Closes #758

As well as adding the worked example, I added a note to **Contract status** on `codelists.md` as that seems like the most proximate place to signpost publishers and users to the example.

Once agreed, we can add this example to the [statusDetails extension documentation](https://extensions.open-contracting.org/en/extensions/statusDetails/master/) for OCDS 1.1.

### CODE REVIEW

**Comment by duncandewhurst** [comment]
@yolile @jpmckinney this worked example only addresses contract suspension, not suspended tenders (#430). Do you think we should have a separate worked example for suspended tenders, or try to have one worked example that applies to both contract suspension and suspended tenders, or can we do without a suspended tenders example given that the concept only came up once, so far?

File: docs/guidance/map/contract_suspension.md
Comment (jpmckinney) [inline docs/guidance/map/contract_suspension.md]: Up to here, it sounds like the contract could just be 'terminatedEarly'. We'll need to explain a bit more about suspension (mainly that the work can be resumed, as I understand).

**Review by jpmckinney — COMMENTED** [review]
In #430, I'm not clear how a process can be suspended/blocked prior to the award stage. 

Besides courts, #430 mentions "blocked by the decision of Deposit Guarantee Fund commission", but we don't have a full example of either. So, we can't do a worked example.

After the award stage, it's common for the signing of a contract to be blocked during a standstill period, where tenderers challenge the award on specific grounds. I added a comment to #1142 about adding a worked example for that scenario.

This PR is ready for a first review.

File: docs/guidance/map/contract_suspension.md
Comment (jpmckinney) [reply docs/guidance/map/contract_suspension.md]: The first paragraph from https://www.upcounsel.com/contract-suspension is clear:

> A contract suspension is the temporary cessation of performance. It's not the same as [... early] termination, which is a permanent cessation of performance.

Comment (duncandewhurst) [reply docs/guidance/map/contract_suspension.md]: Good catch. I've clarified the description in 17c63f726790742d83b69b30c353730c8047a6b9

**Review by mrshll1001 — APPROVED** [review]
I can't spot any typos or errors, and the expected files are changed/introduced.

**Review by jpmckinney — APPROVED** [review]
I fixed conflicts and made one commit.

**Comment by jpmckinney** [comment]
I've created the follow-up issue for the extension: https://github.com/open-contracting/ocds-extensions/issues/168

After merging, please manually close #758. Since 1.2-dev is not the default branch, GitHub will not close it automatically.

**Pull request merged**
